src/widgets/combobox.ts

```diff
@@ -1,5 +1,5 @@
 import { Widget } from './widget';
-import { ElementArrayFinder } from "../wdio";
+import {ElementArrayFinder, ElementFinder} from "../wdio";
```

Comment:
Very Picky one: I would keep the previous formatting on the imports, so we have the same spacing format.

Reply:
Applied